lib/version.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lib
+
+var Version = "0.0.0+was-not-built-correctly"
```

Comment:
Turns out github.com/appc/spec/schema.version is unexported:
https://github.com/appc/spec/blob/master/schema/version.go

On Wed, Oct 21, 2015 at 11:44 AM, Jonathan Boulle <notifications@github.com>
wrote:

> In lib/version.go
> <https://github.com/appc/acbuild/pull/45#discussion_r42665068>:
>
> > +//
> > +// Licensed under the Apache License, Version 2.0 (the "License");
> > +// you may not use this file except in compliance with the License.
> > +// You may obtain a copy of the License at
> > +//
> > +//     http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +// See the License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +package lib
> > +
> > +var Version = "0.0.0+was-not-built-correctly"
>
> Hm, didn't we discuss just having this pinned to
> github.com/appc/spec/version.Version + git?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/acbuild/pull/45/files#r42665068>.
>